test/test.js

```diff
@@ -105,5 +110,20 @@ versions.forEach(function (version) {
         assert(buf.equals(buf2))
       })
     }
+
+    if (cycleTests.includes(version)) {
+      const folder = path.join(__dirname, version)
+      const chunkDump = path.join(folder, 'chunk.dump')
+      const packetData = path.join(folder, 'packet.data')
+      if (fs.existsSync(chunkDump) && fs.existsSync(packetData)) {
```

Comment:
can you remove this check ?
otherwise I'm afraid we will miss that the path are wrong or files are missing because tests will pass even though it didn't run

Reply:
I did it